xbmc/pvr/timers/PVRTimerInfoTag.cpp

```diff
@@ -239,6 +248,43 @@ void CPVRTimerInfoTag::Serialize(CVariant &value) const
     value["state"] = "unknown";
     break;
   }
+
+  switch (m_type)
+  {
+  case PVR_TIMER_TYPE_ONCE_MANUAL:
+    value["type"] = "manual_once";
+    break;
+  case PVR_TIMER_TYPE_SERIE_MANUAL:
+    value["type"] = "manual_serie";
+    break;
+  case PVR_TIMER_TYPE_ONCE_EPG:
+    value["type"] = "epg_once";
```

Comment:
Some of these strings differ from the constants, maybe they should be the same to avoid confusion?